system_tests/logging_.py

```diff
@@ -68,6 +68,33 @@ def test_log_text(self):
         self.assertEqual(len(entries), 1)
         self.assertEqual(entries[0].payload, TEXT_PAYLOAD)
 
+    def test_log_text_w_metadata(self):
+        TEXT_PAYLOAD = 'System test: test_log_text'
+        INSERT_ID = 'INSERTID'
+        SEVERITY = 'INFO'
+        METHOD = 'POST'
+        URI = 'https://api.example.com/endpoint'
+        STATUS = '500'
```

Comment:
No point:  this is an arbitrary status code, set by the caller:  there are no interesting semantics.